portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+// Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+  var max=getMax();
```

Comment:
Let's make this a `const`.  We typically true to use `const` and `let` whenever possible.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Addressed in latest commit.